plugins/module_utils/net_map/networking_mapper.py

```diff
@@ -570,6 +619,33 @@ def __build_network_tool_common(
         return tool_type(*args_list)
 
 
+class NetworkingRoutersMapper:
+    """
+    Handles the mapping of Networking Definition routers section
+    """
+
+    def __init__(self):
+        pass
+
+    def map_routers(
+        self,
+        router_definitions: typing.Dict[str, networking_definition.RouterDefinition],
+    ) -> typing.Dict[str, networking_env_definitions.MappedRouter]:
+        return {
+            router_name: self.__map_single_router(router_name, router_def)
+            for router_name, router_def in router_definitions.items()
+        }
+
+    def __map_single_router(
```

Comment:
If this one is not accessing anything from the instance it can be a static or class method.